src/App.tsx

```diff
@@ -109,17 +113,26 @@ const App = () => {
           <LocalizationProvider dateAdapter={AdapterMoment} adapterLocale="he">
             <ConfigProvider direction="rtl" locale={heIL}>
               <StyledLayout className="main">
-                <LayoutContext>
-                  <SideBar />
-                  <Layout>
-                    <MainHeader />
-                    <StyledContent>
-                      <StyledBody>
-                        <RoutesList />
-                      </StyledBody>
-                    </StyledContent>
-                  </Layout>
-                </LayoutContext>
+
+                <SideBar pages={PAGES} />
+                <Layout>
+                  <StyledContent>
+                    <StyledBody>
+                      <Routes>
+                        <Route path={PAGES[0].key} element={<DashboardPage />} />
+                        <Route path={PAGES[1].key} element={<TimelinePage />} />
+                        <Route path={PAGES[2].key} element={<GapsPage />} />
+                        <Route path={PAGES[3].key} element={<GapsPatternsPage />} />
+                        <Route path={PAGES[4].key} element={<RealtimeMapPage />} />
+                        <Route path={PAGES[5].key} element={<SingleLineMapPage />} />
+                        <Route path={PAGES[6].key} element={<About />} />
+                        <Route path={'/profile'} element={<Profile />} />
+                        <Route path="*" element={<RedirectToDashboard />} />
+                      </Routes>
+                    </StyledBody>
+                  </StyledContent>
+                </Layout>
```

Comment:
I guess you didn't do it on purpose 
anyway, when solving conflicts with the main branch, please prefer the coding style from the main branch
see this -
https://github.com/hasadna/open-bus-map-search/pull/186
what IDE you're using?